CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
+## 0.2.0 (2023-02-24)
+
+- Added Miden Assembly instructions to the Playground
```

Comment:
nit: For clarity, I would adjust this to "Added a reference table of Miden Assembly instructions to the Playground"